dags/fission_experiment_monitoring.py

```diff
@@ -0,0 +1,77 @@
+import datetime
+
+from airflow import models
+from airflow.operators.sensors import ExternalTaskSensor
+from operators.gcp_container_operator import GKEPodOperator
+from utils.gcp import  bigquery_etl_query
+
+
+default_args = {
+    "owner": "akomar@mozilla.com",
+    "start_date": datetime.datetime(2020, 10, 1),
+    "email": ["telemetry-alerts@mozilla.com", "akomar@mozilla.com", "cdowhygelund@mozilla.com"],
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "depends_on_past": False,
+    # If a task fails, retry it once after waiting at least 5 minutes
+    "retries": 1,
+    "retry_delay": datetime.timedelta(minutes=5),
+}
+
+dag_name = "fission_experiment_monitoring"
+
+with models.DAG(
+        dag_name,
+        schedule_interval="0 1 * * *",
```

Comment:
`copy_deduplicate` runs at 0 1 * * *, so we should wait for it to finish and run this one after.

```suggestion
        schedule_interval="0 2 * * *",
```